api/ordinals/src/api/schemas.ts

```diff
@@ -428,6 +428,19 @@ export const Brc20BalanceResponseSchema = Type.Object({
 });
 export type Brc20BalanceResponse = Static<typeof Brc20BalanceResponseSchema>;
 
+export const Brc20TransferableInscriptionsResponseSchema = Type.Object({
+  inscription_number: Type.Integer({ examples: [1095397] }),
+  inscription_id: Type.String({
+    examples: ['ffb9df4532e05fe514765bcbeebb75c38d8f1774a291f9d6f123d68820cc39cdi0'],
+  }),
+  block_height: Type.Integer({ examples: [785143] }),
```

Comment:
I don't think we need `block_height` in this response, can you add `ordinal_number` here instead?